modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -325,4 +320,59 @@ public function get_user_id() {
 		return 'null';
 	}
 
+	/**
+		 * @param $cart_item_key
+		 * @param $product_id
+		 * @param $quantity
+		 * @param $variation_id
+		 * @param $variation
+		 * @param $cart_item_data
+		 */
+		public function capture_add_to_cart( $cart_item_key, $product_id, $quantity, $variation_id, $variation, $cart_item_data ) {
+			if ( ! is_single() ) {
```

Comment:
`is_single()` doesn't seem to work here so we always get a `page_view` even if we've already logged one on the single product page. Any ideas @psealock @timmyc ?